docs/user-guide/ze-use-cics-resources.md

```diff
@@ -3,83 +3,134 @@
+
+   A context menu lists the available actions that can be performed.
+
+3. Select the **Show Attributes** option.
+
+   All attributes and their values open in an **Editor**. Use the filter box at the top of the **Editor** to search for attributes matching the search criteria.  
 
 ![Zowe CICS Explorer Show Attributes](../images/ze-cics/show-attributes.gif)
 
 ## Enabling and disabling
 
-1. Right-click a program, local transaction, or local file to open a pop-up menu that lists the available actions that can be performed.
+1. Expand the **CICS** tree in the **Side Bar** and navigate to a program, local transaction, or local file.
+
+2. Right-click a program, local transaction, or local file.
+
+   A context menu lists the available actions that can be performed.
+
+3. Select the **Disable [CICS resource]** option.
 
-2. Click **Disable [CICS resource]** to disable the resource. A disabled resource is identified by `(Disabled)` text next to its name.
+   The resource is disabled. A disabled resource is identified by a `(Disabled)` indicator next to its name.
 
-When a resource is already disabled, you can re-enable it by clicking **Enable [CICS resource]** in the pop-up menu.
+4. To enable resource, right-click a disabled program, local transaction, or local file and select the **Enable [CICS resource]** option from the context menu to re-enable it.
 
 ![Zowe CICS Explorer Disable and Enable](../images/ze-cics/disable-enable.gif)
 
 ## New copy and phase in
 
-Use the new copy and the phase in actions against a CICS program to get the CICS region to load a fresh copy of the selected program into memory. This could be useful after you edited a COBOL program source and successfully compiled it into a load library and now want to test your change.
+Use the **New Copy** and the **Phase In** actions against a CICS program to get the CICS region to load a fresh copy of the selected program into memory. This could be useful after you edited a COBOL program source and successfully compiled it into a load library and now want to test your change.
 
-The `New copy count` for a program which is greater than zero is shown next to the program item in the CICS resource tree.
+1. Expand the **CICS** tree in the **Side Bar** and navigate to a program.
+
+2. Select the program and click on the **New Copy** icon next to it, or right-click a program and select the **New Copy** option from the context menu.
+
+   The program is applied to the CICS region to use it on the associated transaction, and a pop-up message displays the status of the operation.
+
+   The `New copy count` for a program is shown next to the program in the **CICS** tree.
+
+3. Select the program and click on the **Phase In** icon next to it, or right-click a program and select the **Phase In** option from the context menu.
+
+   The program is applied to the CICS region whether or not the program is actively running, and a pop-up message displays the status of the operation. **[correct?]**
```

Comment:
@adam-wolfe @zFernand0 - please confirm whether this is correct

Reply:
From what is listed in the VSCE README, the "Phase In" action causes the CICS region to load a fresh copy of the selected program into memory. Based on the description, it sounds like the program doesn't have to be actively running in order to get the CICS region to load it:

https://github.com/zowe/cics-for-zowe-client/blob/49aadd944f080d6eeabb3f8e72e599df0f1ebe50/packages/vsce/README.md?plain=1#L269-L271